website/docs/reference/rbac.md

```diff
@@ -104,6 +104,10 @@ You can assign the following permissions on a per-environment level within the p
 
   Lets the user enable and disable toggles within the environment.
 
+- **update variants on environment**
```

Comment:
I know this is what the UI says, but I think we should shorten it. We should also update the UI: there's some inconsistencies right now with how it's used.

```suggestion
- **update variants**
```

Reply:
I think I can also update the message on the UI to say update variants. In other permissions we also prepend "in this environment", do you think that's valuable or should we keep it short? I think the context of the environment it's right there so it's probably not needed

![image](https://user-images.githubusercontent.com/455064/221533701-c2d305cc-de7e-48ca-a902-f1753a17cfd3.png)

Reply:
Yeah, that's my point, exactly 😄  I think for this list, it shouldn't mention "in this environment", so it should also be "enable/disable toggles"

Reply:
I'm making it better for variants: https://github.com/Unleash/unleash/pull/3195/commits/ac017c785f88f65ddfc8dfecb0b8c8d3f778529e but let me know if you have better proposals for the others, this could be a good time to improve them, because it's unfortunate but we do have to run an SQL migration to change the display name

Reply:
Yeah, so my proposal is to leave most unchanged. I would change these, though:
- "Enable/disable toggles in this environment" -> "Enable/disable toggles" (or should it be features? maybe later 🤔 )
- "Approve a change request" -> "Approve change requests"
- "Apply a change request" -> "Apply change requests"

And maybe
- "skip change request process" -> "skip change request process (API-only)" for clarity?
wdyt?

Reply:
I like the proposals, also not adding a lot of unrelated changes is fine, but I agree with your recommendations. Applied!